### PR TITLE
Enable CDS at build time

### PR DESCRIPTION
**Pull request by joe-braley**
Initial attempt to enable CDS during build time for JDK11u. This change was backported up to JDK12. Though we have a desire to ship with CDS enabled out the door. 

This change is heavily inspired from the Makefiles below. Please have a look and provide constructive feedback!
[JDK12u Images.gmk](https://github.com/openjdk/jdk12u/blob/master/make/Images.gmk)
[JDK17u Images.gmk](https://github.com/openjdk/jdk17u/blob/master/make/Images.gmk)

### CODE REVIEW

File: make/Images.gmk [hunk 1]
```diff
@@ -88,6 +88,16 @@ ifeq ($(JLINK_KEEP_PACKAGED_MODULES), true)
   JLINK_JDK_EXTRA_OPTS := --keep-packaged-modules $(JDK_IMAGE_DIR)/jmods
 endif
 
+ifeq ($(BUILD_CDS_ARCHIVE), true)
+  ifeq ($(OPENJDK_TARGET_OS), windows)
+		CDS_ARCHIVE := bin/server/classes.jsa
+		CDS_NOCOOPS_ARCHIVE := bin/server/classes_nocoops.jsa
```
Comment (jmjaffe37) [inline make/Images.gmk:94]: Where/ how is this variable used? It seems as though lines 94 and 97 are the only mentions of this variable in the changed lines

Comment (joe-braley) [reply make/Images.gmk]: Removed!

**Comment by d3r3kk** [comment]
@karianna - we have a backport of the JDK12 sources to enable CDS archives in our JDK 11 builds. We have builds that succeed with this backport, but how do we feel in taking this on? We are not aware of any community desire to have this backported to 11 (yet). We can simply keep this as a 'ms-patches' specific branch, or we can try to push it upstream if that is something the community would desire.

**Comment by karianna** [comment]
> @karianna - we have a backport of the JDK12 sources to enable CDS archives in our JDK 11 builds. We have builds that succeed with this backport, but how do we feel in taking this on? We are not aware of any community desire to have this backported to 11 (yet). We can simply keep this as a 'ms-patches' specific branch, or we can try to push it upstream if that is something the community would desire.

We should send a message to jdk-updates stating that we have a patch which we intend to use for our internal needs but we're not sure if it meets the bar for getting into the updates project and are looking for advice on that.

**Pull request closed without merging**